### PR TITLE
♻️ Replace all references of landing page PoC

### PR DESCRIPTION
**Pull request by jasonBirchall**
Overview
---

## 👀 Purpose

The renaming is part of our efforts to align the application's name with our current operations and objectives, specifically focusing on our integration with GitHub. This change will help maintain consistency and clarity in our tooling and documentation.

## ♻️ What's changed

- All instances of the old name operations-engineering-landing-page-poc have been updated to the new name operations-engineering-join-github.

- The change affects various files across the application, including configuration files, documentation, and any scripts where the old name was referenced.

- I have ensured that the renaming does not affect the functionality and performance of the application.

## 📝 Notes

- Please review the changes for accuracy and completeness.
- Check for any missed references or potential issues that might arise from this renaming.
- Suggestions for further improvement or any concerns are highly welcomed.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -10,11 +10,11 @@ RUN apk add --no-cache --no-progress \
   && apk update \
   && apk upgrade --no-cache --available
 
-WORKDIR /home/operations-engineering-poc-landing-page
+WORKDIR /home/operations-engineering-jon-github
```
Comment (tamsinforbes) [inline Dockerfile:13]:
```suggestion
WORKDIR /home/operations-engineering-join-github
```

Comment (tamsinforbes) [reply Dockerfile:13]: I assume this was deliberate just to check we are reading it 😆

File: join_github_app/templates/components/base.html [hunk 2]
```diff
@@ -11,20 +11,20 @@
 {% extends 'govuk_frontend_jinja/template.html' %}
 
 {% block pageTitle %}
-    Operations Engineering Landing Page PoC
+    Join a Ministry of Justice Github Organisation
```
Comment (tamsinforbes) [inline join_github_app/templates/components/base.html:14]:
```suggestion
    Join a Ministry of Justice GitHub Organisation
```

File: join_github_app/templates/components/base.html [hunk 3]
```diff
@@ -11,20 +11,20 @@
...
 {% endblock %}
 
 {% block head %}
-    <title>Operations Engineering Landing Page PoC</title>
+    <title>Join a Ministry of Justice Github Organisation</title>
```
Comment (tamsinforbes) [inline join_github_app/templates/components/base.html:18]:
```suggestion
    <title>Join a Ministry of Justice GitHub Organisation</title>
```

File: Dockerfile (hunk 1 above)
Comment (jasonBirchall) [inline Dockerfile:13]: Need to replace this with join

File: join_github_app/templates/components/base.html [hunk 4]
```diff
@@ -11,20 +11,20 @@
...
     <!-- Manually set the govuk frontend version in line below and bottom of this file -->
     <link rel="stylesheet" type="text/css"
           href="{{ url_for('static', filename='stylesheets/govuk-frontend-4.3.0.min.css') }}"/>
-    <link rel="canonical" href="https://operations-engineering-landing-page-poc.cloud-platform.service.justice.gov.uk/">
-    <meta property="og:site_name" content="Operations-Engineering Landing Page PoC"/>
-    <meta property="og:title" content="Operations-Engineering Landing Page PoC"/>
+    <link rel="canonical" href="https://operations-engineering-join-github-dev.cloud-platform.service.justice.gov.uk/">
+    <meta property="og:site_name" content="Join a Ministry of Jusitce GitHub Organisation"/>
+    <meta property="og:title" content="Join a Ministry of Jusitce GitHub Organisation"/>
```
Comment (jasonBirchall) [inline join_github_app/templates/components/base.html:24]: Spelling errors.

**Review by tamsinforbes — CHANGES_REQUESTED** [review]
Couple of typos otherwise LGTM

**Review by tamsinforbes — APPROVED** [review]

**Pull request merged**